ui/packages/app/src/api.ts

```diff
@@ -129,7 +129,7 @@ export const fn =
 			}
 			const WS_ENDPOINT = `${ws_protocol}//${ws_host}${ws_path}queue/join`;
 
-			var websocket = new WebSocket(WS_ENDPOINT);
+			var websocket = new WebSocket(WS_ENDPOINT + `?fn=${fn_index}`);
```

Comment:
nit: I think naming the parameter `fn_index` would be more consistent with the rest of the routes/codebase.

Reply:
Sorry I must be missing something but isn't the name already `fn_index`?